api/view/static/openapi.json

```diff
@@ -0,0 +1,268 @@
+{
+  "openapi": "3.0.0",
+  "info": {
+    "description": "Geometry Data Service",
+    "version": "1.0.0",
+    "title": "Geometry Data Service",
+    "license": {
+      "name": "Apache 2.0",
+      "url": "http://www.apache.org/licenses/LICENSE-2.0.html"
+    }
+  },
+  "tags": [
+    {
+      "name": "geometry",
+      "description": "Geometries"
+    },
+    {
+      "name": "search",
+      "description": "Perform search functions"
+    }
+  ],
+  "paths": {
+    "/geometry": {
+      "get": {
+        "tags": [
+          "geometry"
+        ],
+        "summary": "List geometries available",
+        "description": "",
+        "responses": {
+          "200": {
+            "description": "OK",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "type": "string"
+                }
+              },
+              "text/plain": {
+                "schema": {
+                  "type": "string"
+                }
+              },
+              "text/turtle": {
+                "schema": {
+                  "type": "string"
+                }
+              },
+              "application/rdf+xml": {
+                "schema": {
+                  "type": "string"
+                }
+              },
+              "application/ld+json": {
+                "schema": {
+                  "type": "string"
+                }
+              },
+              "text/n3": {
+                "schema": {
+                  "type": "string"
+                }
+              },
+              "application/n-triples": {
+                "schema": {
+                  "type": "string"
+                }
+              },
+              "text/html": {
+                "schema": {
+                  "type": "string"
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+    "/geometry/{dataset_geomId}": {
```

Comment:
My feeling here is that it is just a geomID. Certainly, a geomID might have a datasetID embedded within it in some cases, but as far as the API specification is concerned it is just a single ID and should be described as such. Adding ‘dataset_’ in the doco is exposing an artefact of the only implantation we have so far. Other implementations might construct the geometry-identifiers ways that are not scoped to datasets.

Reply:
reasonable. `dataset_geomId` is the geometryID. we don't necessarily need to specify that there is a geometry ID. however, in the backend, this is how we're expecting things to be implemented to be queried. conceptually, it shouldn't matter though...

Reply:
Where I am coming from is that I don't see why geometries would be scoped to known datasets _a priori_. A single geometry might be used by features in several datasets.

Reply:
agree. i've made that change now.